ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml

```diff
@@ -49,6 +49,28 @@ resources:
       cpu: "1"
       memory: 4Gi
 tests:
+- as: ocp-e2e-ibmcloud-ovn-multi-ppc64le
+  cron: 20 23 * * *
+  steps:
+    cluster_profile: ibmcloud-multi-ppc64le
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: ppc64le
+      BASE_DOMAIN: ocp-dev-ppc64le.com
+      E2E_RUN_TAGS: '@ibmcloud-ipi and @heterogeneous and @network-ovnkubernetes and
+        not @fips'
+      GO_VERSION: 1.20.7
+      JQ_VERSION: "1.6"
+      LOKI_ENABLED: "false"
+      OCP_VERSION: "4.14"
+      RESOURCE_GROUP: ipi-cicd-resource-group
+      SIZE_VARIANT: large
+      TERRAFORM_VERSION: 1.5.5
+      WORKERS: "2"
+      YQ_VERSION: 4.30.5
+    workflow: openshift-e2e-ibmcloud-heterogeneous-upi
+  timeout: 8h0m0s
```

Comment:
I suspect we don't have to change the prior two workflows if we change this one.